index.html

```diff
@@ -81,35 +79,34 @@ <h1>I'm Adeyemi Glad to see you!</h1>
             </p>
           </div>
           <div class="Social-media">
-            <div class="socialText"> 
+            <div class="media-Container"> 
               <div class="socioText">
                 <p>LET'S CONNECT</p>
               </div>
             </div>
             <div class="social-media-icon">
               <li>
```

Comment:
- [ ] As suggested by the previous reviewer, > * [ ]  kindly update your code to make it more semantic and for the best HTML practices : the social icons should be listed inside a list `<ul> & <li>` instead of normal divs, and each social icon should be an `<a>` that contains the link for that icon
>   example:
> 
> ```
> <ul class='...'>
> <li class='...'><a href='...' ><svg>...</svg></a></li>
> <li class='...'><a href='...' ><svg>...</svg></a></li>
> ...
> <li class='...'><a href='...' ><svg>...</svg></a></li>
> </ul>
> ```